astropy/utils/tests/test_data.py

```diff
@@ -903,6 +904,17 @@ def test_find_invalid():
         )
 
 
+def test_get_invalid():
+    """Test can create a file path to an invalid file."""
+    path = get_pkg_data_path("kjfrhgjkla", "hgiulrhgiu")
```

Comment:
Would actually be like this with my suggested changes:
```suggestion
    path = get_pkg_data_path(("kjfrhgjkla", "hgiulrhgiu"))
```

Also need to test non-tuple case:
```python
    path = get_pkg_data_path("kjfrhgjkla")
```